apollo-api/src/commonMain/kotlin/com/apollographql/apollo/api/internal/json/JsonWriters.kt

```diff
@@ -0,0 +1,13 @@
+package com.apollographql.apollo.api.internal.json
+
+inline fun JsonWriter.writeObject(crossinline block: JsonWriter.() -> Unit) {
```

Comment:
Ideally I would want these to be `internal` but since they live in a separate module the cannot. 

They are very handy in making the code more readable and makes manual JSON writing much less error prone.

Reply:
Since the package is marked as `internal`, it should be ok